pkg/gui/custom_commands.go

```diff
@@ -151,10 +153,63 @@ func (gui *Gui) handleCustomCommandKeybinding(customCommand config.CustomCommand
 						return gui.surfaceError(err)
 					}
 
+					return gui.createMenu(title, menuItems, createMenuOptions{showCancel: true})
+				}
+			case "menuFromCommand":
+				f = func() error {
+					// Collect cmd to run from config
+					cmdStr, err := gui.resolveTemplate(prompt.Command, promptResponses)
+					if err != nil {
+						return gui.surfaceError(err)
+					}
+
+					// Collect Filter regexp
+					filter, err := gui.resolveTemplate(prompt.Filter, promptResponses)
+					if err != nil {
+						return gui.surfaceError(err)
+					}
+
+					// Run and save output
+					message, err := gui.GitCommand.RunCommandWithOutput(cmdStr)
+					if err != nil {
+						return gui.surfaceError(err)
+					}
+
+					// Need to make a menu out of what the cmd has displayed
+					candidates := []string{}
+					reg := regexp.MustCompile(filter)
```

Comment:
As the filter is user input there is a change the regex is invalid and i would prefer to get a error message over a panic
```suggestion
					reg, err := regexp.Compile(filter)
					if err != nil {
						return gui.surfaceError(errors.New("unable to parse filter regex, error: " + err.Error()))
					}
```
*Note: you still need to import the `errors` package for this to work*